### ISSUE TITLE
On macOS altool (used for notarization) will no longer supported from "late 2023" onwards

### ISSUE THREAD

**Issue by dyaandys** [Dyalog/ride#954]
Andy saw the following when building for mac_arm:

13:54:31     77	====== 230518 135400 success
13:54:31     78	2023-05-18 13:54:02.612 *** Warning: altool has been deprecated for notarization and starting in late 2023 will no longer be supported by the Apple notary service. You should start using notarytool to notarize your software. (-1030)
13:54:31     79	No errors getting notarization info.

Looks like altool needs to be replaced.  Need to worry about earlier RIDEs too.

For the interpeter(s) see mantis 20574.

**Comment by JasonRivers** [comment]
From the manpage (man notarytool):

```
ALTOOL MIGRATION
     In fall of 2023, altool will no longer be supported for notarization. Use notarytool instead. If you previously invoked altool like this:
         altool --notarize-app -f path/to/app.pkg --primary-bundle-id com.example.myapp --apiKey 7UD13000 --issuerId 6bc36aee-c5c8-11ec-9d64-0242ac120001

     Instead, use notarytool like this:
         notarytool submit path/to/app.pkg --key path/to/AuthKey_7UD13000.p8 --key-id 7UD13000 --issuer 6bc36aee-c5c8-11ec-9d64-0242ac120001 --wait
```

**Comment by e9gille** [comment]
Fixed in #999
